streamlit_folium/__init__.py

```diff
@@ -177,6 +177,26 @@ def _get_feature_group_string(
     return feature_group_string
 
 
+def _get_layer_control_string(
+    control: folium.LayerControl,
+    map: folium.Map,
+) -> str:
+    control._id = "layer_control"
+    control.add_to(map)
+    control.render()
+    control_string = generate_leaflet_string(control, base_id="layer_control")
+    m_id = get_full_id(map)
+    control_string = control_string.replace(m_id, "map_div")
+    control_string = dedent(control_string)
```

Comment:
Not sure it changes anything from a practical standpoint, but doesn't feel like we need to call dedent twice (maybe at all?)

Reply:
It actually does potentially make sense to call it here twice:
1. The first time is in case the leaflet string is indented (probably not, but maybe it is by default for some reason, I can't remember for sure)
2. The second time is to dedent just that newly added string.

You could also do the dedent manually
```
    control_string += (
        """
window.layer_control = layer_control_layer_control;
        """
    )
```
    
But I think manually-unindented strings are ugly in indented code 🤷 

If you just run dedent once, and the original code is not indented, or is indented a different amount, it won't work, because
```
    this code
        window.layer_control ...
        
```
->

```
this code
  window.layer_control ...
```
Instead of 

```
this code
window.layer_control ...
```